exports.login = async (ctx, next) => {
  //   ctx.utils.throwError("500", "手动抛出异常");
  //   console.log(ctx.cloud);
  //   console.log(ctx.db);
  console.log(a); //错误  不存在a变量

  ctx.body = {
    data: "输出",
  };
};
